great_expectations/expectations/core/expect_column_pair_values_to_be_equal.py

```diff
@@ -34,14 +34,23 @@ class ExpectColumnPairValuesToBeEqual(ColumnPairMapExpectation):
     expect_column_pair_values_to_be_equal is a \
     [Column Pair Map Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_pair_map_expectations).
 
-    Args:
-        column_A (str): The first column name
-        column_B (str): The second column name
+    Column Pair Map Expectations are evaluated for a pair of columns and ask a yes/no question about the row-wise relationship between those two columns.
+    Based on the result, they then calculate the percentage of rows that gave a positive answer.
+    If the percentage is high enough, the Expectation considers that data valid.
```

Comment:
```suggestion
   The Expectation considers the data valid when the percentage reaches a high value.
```
How is _high value_ determined? Is this user-defined? More information might be helpful here.